src/test_llm_agent.py

```
@@ -0,0 +1,28 @@
+from intent.llm_agent import LLMIntentAgent
+
+class MockMessages:
+    def create(self, **kwargs):
```

Ruff (0.14.6)

Unused method argument: `kwargs`

(ARG002)